File: PHP/creditCard.php
<?php
require_once('payment.php');
class creditCard extends Payment{
    public $number;
    public $cvv;
    public $date;

    public function __construct($id, $number, $cvv, $date){
        parent __construct($id);
        $this->number=$number;
        $this->cvv=$cvv;
        $this->date=$date;
    }
}
?>

File: Java/CreditCard.java
package Java;

public class CreditCard extends Payment {
    Integer number;
    Integer CVV;
    String date;

    public CreditCard(Integer id, Integer number, Integer CVV, String date){
        super(id);
        this.number=number;
        this.CVV=CVV;
        this.date=date;
    }
    
}
